src/sandstorm/web-session.capnp

```diff
@@ -38,6 +38,11 @@ interface WebSession @0xa50711a14d35a8ce extends(Grain.UiSession) {
   # application's storage was written and may automatically implement etags based on hashing the
   # content.
 
+  const appHeaderPrefix :Text = "x-sandstorm-app-";
```

Comment:
Then again I suppose we're specifying the whitelist itself in lowercased form. OK, then. Bleh, case-insensitivity is the worst.